Mathmagician/Program.cs

```diff
@@ -10,7 +10,64 @@ class Program
     {
         static void Main(string[] args)
         {
-
+            bool running = true;
+            while (running)
+            {
+                List<string> commands = new List<string>() { "Integers", "Primes", "Fibonacci", "even", "odd" };
+                Console.WriteLine("What would you like for me to do?");
+                for (int i = 0; i < commands.Count; i++)
+                {
+                    Console.WriteLine(commands[i]);
+                }
+                Console.Write("> ");
+                string commandInput = Console.ReadLine();
+                Console.WriteLine("How many should I print?");
+                Console.Write("> ");
+                string listAmountInput = Console.ReadLine();
+                int listAmountParsed = Int32.Parse(listAmountInput);
+
+                Console.WriteLine("Cool, I'm going to print {0} {1} numbers.", listAmountInput, commandInput);
+
+
+
+                switch (commandInput)
+                {
+                    case "Integers":

```

Comment:
@morecallan As it is now, this requires the user to type `Integers` exactly. Consider/think of a way to allow for the user to provide case-insensitive responses. Both `Integer` and `integer` should work.